routes/rooms.go

```diff
@@ -56,6 +74,77 @@ func GetEmergencyRoomById(ctx *gin.Context) {
 	ctx.JSON(http.StatusOK, resp)
 }
 
+// GetEmergencyRooms godoc
+// @Summary    get all emergency rooms
+// @Tags       emergency-rooms
+// @Produce    json
+// @Param      page                            query       uint                    false   "0-indexed page number, 0 is assumed when omitted"
+// @Success    200                             {object}    GetMultipleERsResponse
+// @Failure    501                             {object}    HTTPErrorResponse
```

Comment:
The implementation does not yield any 501 anymore, but 400s instead. Please update and generate new swagger documentation.

Reply:
This still applies